components/bio-formats-plugins/src/loci/plugins/in/ImportProcess.java

```diff
@@ -494,13 +492,8 @@ private void initializeFile() throws FormatException, IOException {
     baseReader.setMetadataFiltered(true);
     baseReader.setGroupFiles(!options.isUngroupFiles() || options.isGroupFiles());
     if(options != null && !options.showROIs()){
-        MetadataOptions mo = baseReader.getMetadataOptions();
-        if(mo == null){
```

Comment:
Although this should not affect our own reader stack which is largely extending `FormatHandler`, could the remove of this null pointer check affect e.g. any third party reader that would not define a default `MetadataOptions`?